site/content/docs/5.2/components/scrollspy.md

```diff
@@ -16,10 +16,6 @@ Scrollspy toggles the `.active` class on anchor (`<a>`) elements when the elemen
 
 - As you scroll the "spied" container, an `.active` class is added and removed from anchor links within the associated navigation. Links must have resolvable `id` targets, otherwise they're ignored. For example, a `<a href="#home">home</a>` must correspond to something in the DOM like `<div id="home"></div>`
 
-- Target elements that aren't visible will be ignored and their corresponding nav items will never receive an `.active` class.
```

Comment:
@mdo as you transferred the whole section, maybe we should keep here a reference like :

'- Target elements that aren't visible will be ignored'